onboarding-checklist.md

```diff
@@ -0,0 +1,145 @@
+- [ ] Go through the [FISMA-Ready Github guide](https://github.com/fisma-ready/github)
+to set up your Github account to be added to CISA
+
+### Microsoft Teams chat ###
+
+Once you're set up on Microsoft Teams, we recommend you join the following:
+
+- [ ] [CISA](https://teams.microsoft.com/l/team/19%3aa40fd338ea7a45de994a7c7ce9671fd4%40thread.skype/conversations?groupId=8c6c0b00-7244-46e1-bef9-76ce2a1906f5&tenantId=3ccde76c-946d-4a12-bb7a-fc9d0842354a)
+- [ ] [CISA - VM- Fusion Branch](https://teams.microsoft.com/l/team/19%3a2d4e9f3e321f4ef9a09f4ed94a32105c%40thread.skype/conversations?groupId=2992771f-df53-4097-87dd-75898a5830db&tenantId=3ccde76c-946d-4a12-bb7a-fc9d0842354a)
+
+### Getting to know the VM Fusion Dev Team ###
+
+- [ ] Bookmark the [dev team project board] to track issues and work across
+multiple Github repositories
+- [ ] Bookmark [VM JIRA] to work with the rest of VM Fusion.
+- [ ] In addition to your CISA email address `first.last@cisa.dhs.gov`, you'll
+also get an account on Trio, which is a Google-based account used for
+cross-team and cross-agency collaboration and is generally
+`first.last@trio.dhs.gov`. Trio may take a few days to get set up.
+  - Once you're set up on Trio, visit
+  [chat.google.com](https://chat.google.com) and join applicable rooms.
+- [ ] Make sure you have been added to several [dev team Google Groups] so you
+can participate in team-wide internal communication:
+  - [ ] [ALL Trio users@beta.dhs.gov](https://groups.google.com/a/beta.dhs.gov/d/forum/all)
+  - [ ] [CISA COOL](https://groups.google.com/a/trio.dhs.gov/forum/#!forum/cisa-cool-group)
+  - [ ] [cisagov-github](https://groups.google.com/a/trio.dhs.gov/forum/#!forum/cisagov-github-group)
+  - [ ] [NCATS@beta.dhs.gov](https://groups.google.com/a/beta.dhs.gov/d/forum/ncats)
+  - [ ] [NCATS AWS Support@beta.dhs.gov](https://groups.google.com/a/beta.dhs.gov/d/forum/ncats-aws-support)
+  - [ ] [NCATS Infrastructure Development@beta.dhs.gov](https://groups.google.com/a/beta.dhs.gov/d/forum/ncats-dev)
+- [ ] Add the [NCATS Google Drive folder] to your Google Drive. If you create
+or move a doc there, it'll get the right access permissions for team members
+to be able to view and edit it.
+
+## Role-specific items ##
+
+### Dev Team ###
+
+Useful links:
+
+- [ ] [NCATS Data Dictionary](https://github.com/cisagov/ncats-data-dictionary)
+repo for information about what's stored in the MongoDB database
+- [ ] [CISA COOL AWS Accounts](https://docs.google.com/spreadsheets/d/1It0FIlG7ZxTwrRV-zVYUAMw18L6ZstQiAYT7vGYf5VE/edit#gid=2069139012)
+sheet for AWS accounts reference
+- [ ] [Performance Plans](https://docs.google.com/spreadsheets/d/1UaGI8u70CXOGBtvJFQ85vCkPbqyDh5PVws0Xu893piw/edit?ts=5e558417&pli=1#gid=474573185)
+sheet, until performance plans are individualized again (sometime after
+June 2020)
+- [ ] [COOL Wiki](https://github.com/cisagov/cool-system/wiki/)
+- [ ] Set up your development environment on your CISA-provided Mac laptop
+(Link TBD)
```

Comment:
I think you created the source for this link, so it could be added now.

Reply:
Resolved in 802d2dd2215abd0eda70209fe993f0b5d8227ad4